dashboard/app/models/lesson.rb

```diff
@@ -252,8 +252,9 @@ def summarize(include_bonus_levels = false, for_edit: false)
         levels: cached_levels.map {|sl| sl.summarize(false, for_edit: for_edit)},
         description_student: render_codespan_only_markdown(I18n.t("data.script.name.#{script.name}.lessons.#{key}.description_student", default: '')),
         description_teacher: render_codespan_only_markdown(I18n.t("data.script.name.#{script.name}.lessons.#{key}.description_teacher", default: '')),
-        unplugged: display_as_unplugged, # TODO: Update to use unplugged property
-        lessonEditPath: edit_lesson_path(id: id)
+        unplugged: script.is_migrated ? unplugged : display_as_unplugged, # TODO: Update to use unplugged property
+        lessonEditPath: edit_lesson_path(id: id),
+        activities: lesson_activities.map(&:summarize)
```

Comment:
Is this a new line you added or is something funky going on with git?

Reply:
I did add activities, they weren't being provided before

Reply:
I see now that it is used in progress.js. I think the suggestion in that file will make it so you don't actually need to pass this down anymore.